Backpack-SwiftUI/Slider/Classes/BPKRangeSlider.swift

```diff
@@ -167,7 +180,8 @@ public struct BPKRangeSlider: View {
             .padding(.bottom, flareHeight)
             .background(.coreAccentColor)
             .clipShape(LabelFlareShape(flareHeight: flareHeight))
-            .padding(.bottom, thumbSize + BPKSpacing.sm.value)
+            .frame(height: thumbSize)
+            .offset(y: -(thumbSize + flareHeight + BPKSpacing.sm.value))
```

Comment:
this makes the thumb label be teh same physical size as the thubs themselves, while keeping the correct looks. Also, moving it up with Offset instead of padding so that the thumbLabels dont push the view down whne they appear